flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/ShardConsumer.java

```diff
@@ -22,21 +22,29 @@
 import org.apache.flink.streaming.connectors.kinesis.config.ConsumerConfigConstants;
```

Comment:
@xiaolong-sn I was not expecting any changes to `ShardConsumer`, once you move `LAZY` to `KinesisDataFetcher` this should be unchanged

Reply:
Solved